### PR TITLE
[Grouping&Mapping] Loading and adding groups toolbar fix

### PR DESCRIPTION
**Pull request by arinsasmal**

### CODE REVIEW

File: common/changes/@itwin/grouping-mapping-widget/loading-groups-toolbar-change_2022-08-15-16-15.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/grouping-mapping-widget",
+      "comment": "Improved button wrapping for the widget",
```
Comment (Guillar1) [inline common/changes/@itwin/grouping-mapping-widget/loading-groups-toolbar-change_2022-08-15-16-15.json:5]:
```suggestion
      "comment": "Improved button wrapping for the widget in Group view",
```

File: common/changes/@itwin/grouping-mapping-widget/loading-groups-toolbar-change_2022-08-11-15-52.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
```
Comment (Guillar1) [inline common/changes/@itwin/grouping-mapping-widget/loading-groups-toolbar-change_2022-08-11-15-52.json:1]: This shouldn't be here. Bad merge?

File: common/changes/@itwin/grouping-mapping-widget/loading-groups-toolbar-change_2022-08-15-16-15.json (hunk 1 above)
Comment (arinsasmal) [reply common/changes/@itwin/grouping-mapping-widget/loading-groups-toolbar-change_2022-08-15-16-15.json:5]: Done

File: common/changes/@itwin/grouping-mapping-widget/loading-groups-toolbar-change_2022-08-11-15-52.json (hunk 2 above)
Comment (arinsasmal) [reply common/changes/@itwin/grouping-mapping-widget/loading-groups-toolbar-change_2022-08-11-15-52.json]: Done

**Pull request merged**